### ISSUE TITLE
ns.hackAnalyzeThreads giving wrong reslut?

### ISSUE THREAD

**Issue by schnapper79** [bitburner-official/bitburner-src#446]
i use `ns.hackAnalyzeThreads(server, money)` in a monitor script, where server in this case is ecorp and money the maximum money.

```
 while (true) {
        const server = flags._[0];
        let money = ns.getServerMoneyAvailable(server);
        if (money === 0) money = 1;
        const maxMoney = ns.getServerMaxMoney(server);
        const minSec = ns.getServerMinSecurityLevel(server);
        const sec = ns.getServerSecurityLevel(server);
        ns.clearLog(server);
        ns.print(`${server}:`);
        ns.print(` $_______: ${ns.nFormat(money, "$0.000a")} / ${ns.nFormat(maxMoney, "$0.000a")} (${(money / maxMoney * 100).toFixed(2)}%)`);
        ns.print(` security: +${(sec - minSec).toFixed(2)}`);
        ns.print(` hack____: ${ns.tFormat(ns.getHackTime(server))} (t=${Math.ceil(ns.hackAnalyzeThreads(server, money))})`);
}
```

Result: 
```
ecorp:
 $_______: $2.866t / $2.866t (100.00%)
 security: +0.00
 hack____: 58 seconds (t=1)
```
Now, when running a hack(ecorp) with one thread, i get just some little money.

analyze on ecorp shows:
```
[ecorp ~/]> analyze 
Analyzing system...
[||||||||||||||||||||||||||||||||||||||||||||||||||]
ecorp: 
Organization name: ECorp
Root Access: YES
Can run scripts on this host: NO
RAM: 0.00GB
Backdoor: NO
Required hacking skill for hack() and backdoor: 1074
Server security level: 14.414
Chance to hack: 100.00%
Time to hack: 58.999 seconds
Total money available on server: $2.866t
```

**Comment by Snarling** [comment]
Looks like this is a new issue in 2.3.0, seems to be returning the reciprocal of how many threads you actually need.

**Comment by Snarling** [comment]
Yeah that was my bad, I messed up some parentheses. Fixed in dadf42b and is live on the [web dev build ](https://bitburner-official.github.io/bitburner-src/)
